### PR TITLE
Add maintainers for jms-messaging plugin

### PR DESCRIPTION
**Pull request by olivergondza**
# Description

Add @olivergondza and @pjanouse as maintainers of https://github.com/jenkinsci/jms-messaging-plugin. @scoheb, please approve.

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by pjanouse** [comment]
For an unknown reason I'm cited as "pajasoft" in developers section for plugins although I've signed infra-cla as "pjanouse" - see for ex. `node-sharing-orchestrator.yml`

**Review by scoheb — APPROVED** [review]

**Comment by slide** [comment]
`User name not known to Artifactory: pjanouse`

The username in this file should be the Jenkins infra username (Jira login, etc) not the GH username.

**Comment by pjanouse** [comment]
pjanouse -> pajasoft

File: permissions/plugin-jms-messaging.yml [hunk 1]
```diff
@@ -6,3 +6,5 @@ paths:
 developers:
 - "scoheb"
 - "ggallen"
+- "olivergondza"
+- "pjanouse"
```
Comment (slide) [inline permissions/plugin-jms-messaging.yml:10]:
```suggestion
- "pajasoft"
```

**Pull request merged**